DQM/SiStripMonitorTrack/src/SiStripMonitorTrack.cc

```diff
@@ -1276,6 +1281,13 @@ bool SiStripMonitorTrack::clusterInfos(
       if(noise == 0.0)
 	LogDebug("SiStripMonitorTrack") << "Module " << detid << " in Event " << eventNb << " noise " << noise << std::endl;
     }
+    if (TkHistoMap_On_ && (flag == OnTrack)) {
```

Comment:
It is bool type. If I add the ! = nullptr, I can't compile it and get the following error 

/afs/cern.ch/work/x/xuchen/public/CMSSW_9_2_4/src/DQM/SiStripMonitorTrack/src/SiStripMonitorTrack.cc:1284:27: error: invalid operands of types 'bool' and 'std::nullptr_t' to binary 'operator!='
     if (TkHistoMap_On_ != nullptr && (flag == OnTrack)) {